### ISSUE TITLE
Collection.editors & Collection.viewers shouldn't be just an array of strings

### ISSUE THREAD

**Issue by Ratatinator97** [Pictalk-speech-made-easy/Pictalk-API-v2#11]
@UnlikelyBuddy1 : 
Having raw strings of usernames is clunky... Not consistent and not optimized.
Our ORM isn't optimized for such a kind of operations and we need to have raw SQL queries on Sharing.
We need to use the userIds in the viewers and editors.
The frontend **should not be affected**

**Comment by UnlikelyBuddy1** [comment]
Yeah that's true but for the time being the current solution is OK and we would have to migrate
